.github/workflows/_docker-image.yaml

```
@@ -101,16 +122,55 @@ jobs:
-          platforms: ${{ inputs.platforms }}
           builder: ${{ steps.buildx.outputs.name }}
+          image_name: ${{ needs.prepare.outputs.image_name }}
+          alter_image_name: ${{ needs.prepare.outputs.alter_image_name }}
+          primary_tag: ${{ needs.prepare.outputs.primary_tag }}
+          secondary_tags: ${{ needs.prepare.outputs.secondary_tags }}
       - name: Scan the Docker image
         if: startsWith(github.ref, 'refs/tags/')
         uses: ./.github/actions/scan-docker-image
         with:
-          image_ref: "${{ steps.build_and_publish.outputs.IMAGE_NAME }}:${{ steps.build_and_publish.outputs.PRIMARY_TAG }}"
+          image_ref: "${{ steps.build_and_publish.outputs.IMAGE_NAME }}:${{ steps.build_and_publish.outputs.PRIMARY_TAG }}-${{ matrix.arch }}"
```

actionlint (1.7.7)

property "image_name" is not defined in object type {extra_tags: string; platforms: string}

(expression)